pinot-query-runtime/src/test/resources/queries/NumericTypes.json

```diff
@@ -0,0 +1,117 @@
+{
+  "smallint": {
+    "psql": "8.1.1",
+    "ignored": true,
+    "comment": "not supported"
+  },
+  "integers": {
+    "comment": "we don't support BIGINT notation",
```

Comment:
we don't support BIGINT in schema definition. and we don't support LONG in SQL syntax.